Lib/test/test_ssl.py

```diff
@@ -988,6 +996,7 @@ def test_load_verify_cadata(self):
             ctx.load_verify_locations(cadata=b"broken")
 
 
+    @unittest.skipIf(not _can_decode_unicode(u'dhpäräm.pem'), "cannot decode unicode path")
```

Comment:
I would prefer to put the encoding test inside the function. You can use:

except UnicodeEncodeError: self.skipTest("filesystem encoding cannot encode non-ASCII path")

> u'dhpäräm.pem'

Pleaes don't repeat hardcoded strings. Instead: write the string into a variable (in the function, no need for a global) and reuse it.